doc/user/layouts/partials/head.html

```diff
@@ -86,6 +86,20 @@
   });
 </script>
 
+<script
+  async
+  src="https://widget.kapa.ai/kapa-widget.bundle.js"
+  data-website-id="8afe5064-c908-44bd-8cd9-805d9e740fb9"
+  data-project-name="Materialize"
+  data-project-color="#472E85"
+  data-project-logo="https://avatars.githubusercontent.com/u/47674186?s=200&v=4"
+  data-modal-title="Materialize AI"
+  data-modal-disclaimer="This a custom LLM for Materialize with access to developer docs, guides and blog posts. Answers may be inaccurate. Consult the official Materialize documentation to verify any answers provided by Materialize AI."
```

Comment:
```suggestion
  data-modal-disclaimer="This is a custom LLM for Materialize with access to developer docs, guides and blog posts. Answers may be inaccurate. Check the official Materialize documentation to verify any answers provided by Materialize AI."
```

Reply:
Thanks, typo fixed. I prefer "consult" to "check" though. To my ear, "check" implies that something might be wrong with what you're checking, while "consult" implies that who/what you're consulting with is trustworthy. So "consult the docs to check the AI" makes it clear that the docs are trustworthy while the AI is not, but "check the docs to verify (check) the AI" is less clear about which is trustworthy and which is suspect.

Reply:
Cool with keeping the original.